pkgs/development/python-modules/crytic-compile/default.nix

```diff
@@ -0,0 +1,32 @@
+{ buildPythonPackage
+, fetchPypi
+, isPy27
+, lib
+, pysha3
+}:
+
+buildPythonPackage rec {
+  pname = "crytic-compile";
+  version = "0.1.9";
+  disabled = isPy27;
```

Comment:
What is the actual python version that doesn't work? e.g does python3.5 work?

Reply:
it doesn't compile on python 2

Reply:
A lot of packages will require a specific python version (e.g. https://github.com/Qiskit/qiskit-terra/blob/837e79b0396609a5677fd6d42c1f495d3ee62858/setup.py#L107, I would write ``disabled = pythonOlder "3.6";``)
It looks like ``crytic-compile`` requires the same thing.